README.md

```diff
@@ -169,6 +169,127 @@ sudo yum install python3-devel
 sudo yum install python-devel
 ```
 
+# Docker
+
+## Building the [atk-base](docker/atk-base/Dockerfile) Docker image
+
+If you have [Docker](https://docs.docker.com/get-docker/) installed, you can build the base Docker image for the ATK:
+
+``` shell
+git clone https://github.com/BeamIO-Inc/algorithm_toolkit.git
+cd algorithm_toolkit
+docker build -t atk-base -f ./docker/atk-base/Dockerfile .
+```
+
+This is a very light weight Ubuntu based Docker image, that contains the algorithm toolkit, as well as the latest version of [Miniconda](https://docs.conda.io/en/latest/miniconda.html) with Python 3 installed.
+
+## Running the atk-base Docker image we just built (Optional)
```

Comment:
I would maybe leave this out - not really useful for an average user (and may confuse them), and a poweruser will already know this. Maybe instead include a quick blurb about using it as a base for atk projects (like you already have), and a link to the Docker docs about how that works?

Reply:
Will do